host/cluster-down.yaml

```diff
@@ -185,6 +185,48 @@ spec:
         collectorName: "journalctl-dmesg"
         command: "journalctl"
         args: ["--dmesg", "--no-pager", "-S", "7 days ago"]
+    # Logs for K8s Control Plane
+    - run:
+        collectorName: "crictl-logs-apiserver"
+        command: "sh"
+        args: ["-c", "crictl logs $(crictl ps -a --name apiserver -l | awk 'NR!=1 {print $1}') 2>&1"]
```

Comment:
`NR!=1` is that to skip the first line where the headers are printed?

Reply:
otherwise looking pretty good to me

Reply:
> NR!=1 is that to skip the first line where the headers are printed?

It is

Reply:
How about replacing `| awk 'NR!=1 {print $1}'` with `--quite` that way this works in those rare cases where `awk` does not exist?

The command would look like `crictl logs $(crictl ps -a --name apiserver -l --quiet`